src/components/GameDetail/index.jsx

```diff
@@ -0,0 +1,96 @@
+import { useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
+import axios from "axios";
+import parse from "html-react-parser";
+import Skeleton, { SkeletonTheme } from "react-loading-skeleton";
+
+import "react-loading-skeleton/dist/skeleton.css";
+import "./index.css";
+
+export default function GameDetail() {
+  const { id: gameId } = useParams();
+  const [gameData, setGameData] = useState({});
+  const {
+    name,
+    background_image,
+    description,
+    genres,
+    developers,
+    released,
+    publishers,
+    website,
+  } = gameData;
+
+  const options = {
+    method: "GET",
+    url: `https://rawg-video-games-database.p.rapidapi.com/games/${gameId}`,
+    params: { key: import.meta.env.VITE_RAWG_KEY },
+    headers: {
+      "X-RapidAPI-Key": import.meta.env.VITE_RAPID_API_KEY,
+      "X-RapidAPI-Host": "rawg-video-games-database.p.rapidapi.com",
+    },
+  };
+
+  useEffect(() => {
+    (async function () {
+      try {
+        const response = await axios.request(options);
+        console.log(response.data);
+        setGameData(response.data);
+      } catch (error) {
+        console.log(error);
+      }
+    })();
+  }, []);
+  return (
+    <SkeletonTheme baseColor="#171a21" highlightColor="#040f14">
+      <div className="game-detail">
+        <h1 className="game-title">{name || <Skeleton width="400px" />}</h1>
+        <div className="game-detail-container">
+          <div className="game-detail-left">
+            {background_image ? (
```

Comment:
instead of relaying on data property, you should maintain the loading state to show the skeleton.